clar2wasm/tests/wasm-generation/control_flow.rs

```diff
@@ -24,16 +24,12 @@ fn begin_strategy() -> impl Strategy<Value = (String, PropValue, bool)> {
                 expressions.push(' ');
             }
 
-            expressions.push_str(&s);
+            expressions.push_str(&format!("{v}"));
         }
 
         let last_value = values.last().unwrap().clone();
```

Comment:
You could simplify this function a lot with the usage of [`split_last`](https://doc.rust-lang.org/std/primitive.slice.html#method.split_last)

Reply:
Simplify code a bit, but repetition of 2 statements. Fixed though